contrib/fd-plugins/bareos_tasks/pgsql/BareosFdPgSQLClass.py

```diff
@@ -27,13 +27,13 @@ class TaskQueryDatabase(TaskProcess):
 
     def __init__(self, psql=None, pg_user=None):
         self.run_as_user = pg_user
-        psql_options = 'postgres --expanded --no-align'
+        psql_options = '--expanded --no-align --no-psqlrc'
```

Comment:
In my test, specifying a database is required for the psql command. Therefore the default database postgres is given as command. Does it work otherwise in our environment?

Reply:
It worked perfectly without it, the original code didn't have it either.

The --no-psqlrc is definitely an enhancement (which I assume cannot cause any harm), since we have a global psqlrc which breaks the backups otherwise

Reply:
PS: if you want to specify the database, I would use -d postgres explicitly, instead of specifying without -d. But I'm not sure why we'd need this in our case. Also if we'd like to connect as another user, in theory we could have no access to postgres db

Reply:
Check the docs: https://www.postgresql.org/docs/15/libpq-connect.html

![image](https://github.com/bareos/bareos/assets/5538957/7eb64b6c-e1c1-430d-b8c6-7993e764a2ea)

So this is really default behavior to use the same db name as the connecting user! :)

Reply:
I see.
I've now added my commits to this PR (systemtest and a bit of cleanup in the plugin).
I'd like to merge this PR. For this, you should squash your commits into one (`git rebase -i origin/master` ...)
Also the commit comment should be something like:
```
contrib bareos_tasks.pgsql: fixes

Fix reading stderr and encoding issues.
```
Also you might want to add your name to the AUTHORS file.

Reply:
Do I still need to do something? If it could be avoided, it would be better since I'm working on a different project and I'm not extremely acquainted with git either :)